ansible/roles/servers/files/grafana/dashboards/default.json

```diff
@@ -531,56 +531,88 @@
       },
       {
         "cacheTimeout": null,
+        "colorBackground": false,
+        "colorValue": false,
+        "colors": [
+          "#299c46",
+          "rgba(237, 129, 40, 0.89)",
+          "#d44a3a"
+        ],
         "datasource": "Graphite-Stats",
+        "format": "none",
+        "gauge": {
+          "maxValue": 100,
+          "minValue": 0,
+          "show": false,
+          "thresholdLabels": false,
+          "thresholdMarkers": true
+        },
         "gridPos": {
           "h": 4,
           "w": 7,
           "x": 0,
           "y": 25
         },
         "id": 2,
+        "interval": null,
         "links": [],
-        "options": {
-          "fieldOptions": {
-            "calcs": [
-              "sum"
-            ],
-            "defaults": {
-              "decimals": null,
-              "mappings": [],
-              "max": 100,
-              "min": 0,
-              "thresholds": [
-                {
-                  "color": "#7EB26D",
-                  "value": null
-                },
-                {
-                  "color": "#E24D42",
-                  "value": 2
-                }
-              ],
-              "unit": "none"
-            },
-            "override": {},
-            "values": false
+        "mappingType": 1,
+        "mappingTypes": [
+          {
+            "name": "value to text",
+            "value": 1
           },
-          "orientation": "auto",
-          "showThresholdLabels": false,
-          "showThresholdMarkers": true
+          {
+            "name": "range to text",
+            "value": 2
+          }
+        ],
+        "maxDataPoints": 100,
+        "nullPointMode": "connected",
+        "nullText": null,
+        "options": {},
+        "pluginVersion": "6.4.3",
+        "postfix": "",
+        "postfixFontSize": "50%",
+        "prefix": "",
+        "prefixFontSize": "50%",
+        "rangeMaps": [
+          {
+            "from": "null",
+            "text": "N/A",
+            "to": "null"
+          }
+        ],
+        "sparkline": {
+          "fillColor": "rgba(31, 118, 189, 0.18)",
+          "full": false,
+          "lineColor": "rgb(31, 120, 193)",
+          "show": false,
+          "ymax": null,
+          "ymin": null
         },
-        "pluginVersion": "6.4.2",
```

Comment:
should this be replaced with 6.4.3 rather than deleted? I ask because of the change at line 80.

Reply:
Not sure about this. Workflow: when I make a change to the dashboard UI, I just re-export the dashboard and commit it to the api call JSON. If it turns out there's something missing, we can open this as an issue, but doesn't seem to be evident so far.